.github/workflows/build.yml

```diff
@@ -0,0 +1,90 @@
+on:
+  - pull_request
```

Comment:
Originally `gh-pages` branch was excluded from triggering tests. I think it's doable with actions as well.

Reply:
done.